src/components/Tables/TableHead.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <tr>
-    <th v-for="column in columns" :key="column.id">{{ column.label }}</th>
+    <th v-for="column in columns" :key="column.label">{{ column.label }}</th>
```

Comment:
here it was `column.id`. But the columns don't have id property, so it should be `column.key`